services/ui-src/src/verbiage/errors.ts

```diff
@@ -14,6 +14,7 @@ export const validationErrors = {
   INVALID_URL: "Response must be a valid hyperlink/URL",
   INVALID_DATE: "Response must be a valid date",
   INVALID_END_DATE: "End date can't be before start date",
+  NON_POSITIVE_NUMBER: "Response must be equal to or greater than one",
```

Comment:
Comment: It's slightly irksome that this message indicates the value must be at least one, whereas our validation actually checks that the value is greater than zero. If we ever apply this validation to a field where 0.5 would be acceptable, we will need to adjust the message somehow.

Reply:
@benmartin-coforma Yea reading it back it does sound a little more complicated than "greater than zero". And making another message for decimals may be overkill. I'll circle back to Megan, Kim & David and see if we can switch it.

Reply:
@benmartin-coforma There good with '"Response must be a number greater than zero".'